portfolio/src/main/webapp/script.js

```diff
@@ -86,17 +124,48 @@ function getClosestTag(index, source){
     }
 
     var startIndex = index;
+  
+    return source.substring(startIndex, endIndex);
+}
+
+/**
+ * Parses through html tag to get id
+ */
+function getTags(array, source){
+    var retArr = [];
+    console.log("array" + array);
+    for (var i = 0; i < array.length; i++){
+        retArr.push(getClosestTag(array[i], source)); 
+    }
 
-    return source.substring(startIndex, endIndex)
+    return retArr;
 }
 
 /**
-  * Gets HTML id from given word
-  */
+ * Gets the indices of matches
+ */
+function getIndicesOf(small, big) {
```

Comment:
Give these variables more descriptive names and a better function header. This function is very ambiguous to me.